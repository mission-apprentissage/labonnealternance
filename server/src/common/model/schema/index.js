import { apiCallSchema } from "./ApiCall.js"
import { applicationSchema } from "./applications.js"
import { appointmentSchema } from "./appointment.js"
import { credentialSchema } from "./credentials.js"
import { diplomesMetiersSchema } from "./diplomesmetiers.js"
import { domainesMetiersSchema } from "./domainesmetiers.js"
import { etablissementSchema } from "./etablissement.js"
import { formulaireSchema } from "./formulaire.js"
import { bonneBoiteSchema } from "./lbb/bonneBoite.js"
import { emailBlacklist } from "./lbb/emailBlacklist.js"
import { geoLocationSchema } from "./lbb/geoLocations.js"
import { opco } from "./lbb/opco.js"
import { mnaFormationSchema } from "./mnaFormation/mnaFormation.js"
import { offreSchema } from "./offre.js"
import { optoutSchema } from "./optout.js"
import { parcoursupEtablissementStatSchema } from "./parcoursupEtablissementStat.js"
import { userRecruteurSchema } from "./recruteur/users.js"
import { referentielOpcoSchema } from "./referentielOpco.js"
import { userSchema } from "./user.js"
import { widgetParameterSchema } from "./widgetParameter.js"

export default {
  domainesMetiersSchema,
  diplomesMetiersSchema,
  applicationSchema,
  apiCallSchema,
  mnaFormationSchema,
  bonneBoiteSchema,
  geoLocationSchema,
  emailBlacklist,
  opco,
  userSchema,
  appointmentSchema,
  widgetParameterSchema,
  etablissementSchema,
  parcoursupEtablissementStatSchema,
  formulaireSchema,
  offreSchema,
  referentielOpcoSchema,
  optoutSchema,
  userRecruteurSchema,
  credentialSchema,
}
