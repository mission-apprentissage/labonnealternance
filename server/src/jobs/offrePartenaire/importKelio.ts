import { importFromUrlInXml } from "./importFromUrlInXml"

export const importKelio = async () => {
  await importFromUrlInXml({ destinationCollection: "raw_kelio", url: "plop", offerXmlTag: "job" })
}
