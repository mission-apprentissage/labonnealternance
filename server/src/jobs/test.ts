import { ObjectId } from "bson"
import { IApplicant, IApplication, ZApplicant } from "shared"

import { getDbCollection } from "@/common/utils/mongodbUtils"

import { asyncForEach } from "../common/utils/asyncUtils"

type ApplicationAggregate = Pick<IApplication, "applicant_email" | "applicant_first_name" | "applicant_last_name" | "applicant_phone" | "_id">

export const migrationTest = async () => {
  // console.log(extensions.telephone.safeParse("0690760552"))
  // return
  const applications = (await getDbCollection("applications")
    .find({}, { projection: { applicant_first_name: 1, applicant_last_name: 1, applicant_email: 1, applicant_phone: 1, _id: 1 } })
    .toArray()) as ApplicationAggregate[]

  const stat = { error: 0, success: 0, total: applications.length }

  await asyncForEach(applications, async (application) => {
    const { applicant_first_name, applicant_email, applicant_last_name, applicant_phone, _id } = application
    const now = new Date()
    const applicant: IApplicant = {
      _id: new ObjectId(),
      firstname: applicant_first_name,
      lastname: applicant_last_name,
      phone: applicant_phone,
      email: applicant_email,
      last_connection: null,
      createdAt: now,
      updatedAt: now,
    }
    const validation = ZApplicant.safeParse(applicant)
    if (!validation.success) {
      stat.error++
      console.log({ applicant, err: validation.error.errors })
    } else {
      stat.success++
    }
  })
  console.log(stat)
}
