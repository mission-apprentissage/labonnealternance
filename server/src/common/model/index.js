import mongoosePaginate from "mongoose-paginate"
import { getElasticInstance, mongoosastic } from "../esClient/index.js"
import schema from "../model/schema/index.js"
import { mongooseInstance } from "../mongodb.js"

const createModel = (modelName, descriptor, options = {}) => {
  const schema = new mongooseInstance.Schema(descriptor)

  if (options.esIndexName) {
    schema.plugin(mongoosastic, { esClient: getElasticInstance(), index: options.esIndexName })
  }

  if (options.esIndexName || options.paginate) {
    schema.plugin(mongoosePaginate)
  }

  if (options.createMongoDBIndexes) {
    options.createMongoDBIndexes(schema)
  }

  return mongooseInstance.model(modelName, schema)
}

const DomainesMetiers = createModel("domainesmetiers", schema.domainesMetiersSchema, {
  esIndexName: "domainesmetiers",
})
const FormationCatalogue = createModel("formationcatalogues", schema.mnaFormationSchema, {
  esIndexName: "formationcatalogues",
})
const DiplomesMetiers = createModel("diplomesmetiers", schema.diplomesMetiersSchema, {
  esIndexName: "diplomesmetiers",
})
const ApiCalls = createModel("apicalls", schema.apiCallSchema)
const Application = createModel("applications", schema.applicationSchema, {
  paginate: true,
})
const SourceFormations = createModel("sourceformations", schema.sourceFormationsSchema)
const GeoLocation = createModel("geolocation", schema.geoLocationSchema)
const EmailBlacklist = createModel("emailblacklist", schema.emailBlacklist)
const Opco = createModel("opco", schema.opco)
const BonnesBoites = createModel("bonnesboites", schema.bonneBoiteSchema, {
  esIndexName: "bonnesboites",
})
const User = createModel("user", schema.userSchema, { paginate: true })
const Appointment = createModel("appointment", schema.appointmentSchema, { paginate: true })
const WidgetParameter = createModel("widgetParameter", schema.widgetParameterSchema, { paginate: true })
const Etablissement = createModel("etablissement", schema.etablissementSchema, { paginate: true })
const ParcoursupEtablissementStat = createModel("parcoursupEtablissementStat", schema.parcoursupEtablissementStatSchema, { paginate: true })
const Formulaire = createModel("formulaire", schema.formulaireSchema, { esIndexName: "formulaires", paginate: true })
const Credential = createModel("credentials", schema.credentialSchema)
const Offre = createModel("offre", schema.offresSchema)
const Optout = createModel("optout", schema.optoutSchema, { paginate: true })
const ReferentielOpco = createModel("referentielOpco", schema.referentielOpcoSchema)
const UserRecruteur = createModel("userRecruteur", schema.userRecruteurSchema, { paginate: true })
const AppointmentDetailed = createModel("appointmentDetailed", schema.appointmentSchema)

export {
  DomainesMetiers,
  FormationCatalogue,
  DiplomesMetiers,
  ApiCalls,
  Application,
  SourceFormations,
  GeoLocation,
  EmailBlacklist,
  Opco,
  BonnesBoites,
  User,
  Appointment,
  WidgetParameter,
  Etablissement,
  ParcoursupEtablissementStat,
  Formulaire,
  Credential,
  Offre,
  Optout,
  ReferentielOpco,
  UserRecruteur,
  AppointmentDetailed,
}
