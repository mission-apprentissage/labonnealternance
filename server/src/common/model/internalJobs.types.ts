import { ObjectId } from "mongodb"

import { CronName } from "@/jobs/jobs"

/*
  name: Le nom de la tâche
  type: Type du job simple ou cron
  status: Statut courant du job
  sync: Si le job est synchrone
  payload: La donnée liéé à la tâche
  output: Les valeurs de retours du job
  cron_string: standard cron string exemple: '*\/2 * * * *'
  scheduled_for: Date de lancement programmée
  started_at: Date de lancement
  ended_at: Date de fin d'execution
  updated_at: Date de mise à jour en base de données
  created_at: Date d'ajout en base de données
*/

interface IInternalJobsSimple {
  _id: ObjectId
  name: string
  type: "simple"
  status: "pending" | "will_start" | "running" | "finished" | "blocked" | "errored"
  sync: boolean
  payload: Record<string, any>
  output?: Record<string, unknown> | undefined
  scheduled_for: Date
  started_at?: Date
  ended_at?: Date
  updated_at: Date
  created_at: Date
}

interface IInternalJobsCron {
  _id: ObjectId
  name: CronName
  type: "cron"
  status: "pending" | "will_start" | "running" | "finished" | "blocked" | "errored"
  sync: boolean
  cron_string: string
  scheduled_for: Date
  updated_at: Date
  created_at: Date
}

interface IInternalJobsCronTask {
  _id: ObjectId
  name: CronName
  type: "cron_task"
  status: "pending" | "will_start" | "running" | "finished" | "blocked" | "errored"
  sync: boolean
  scheduled_for: Date
  started_at?: Date
  ended_at?: Date
  updated_at: Date
  created_at: Date
}

type IInternalJobs = IInternalJobsSimple | IInternalJobsCron | IInternalJobsCronTask

export type { CronName, IInternalJobs, IInternalJobsCron, IInternalJobsCronTask, IInternalJobsSimple }
