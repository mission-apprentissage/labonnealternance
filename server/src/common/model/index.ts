import { captureException } from "@sentry/node"

import { db, mongooseInstance } from "@/common/mongodb"

import { logger } from "../logger"

import AnonymizedUser from "./schema/anonymizedUsers/anonymizedUsers.schema"
import ApiCalls from "./schema/apiCall/apiCall.schema"
import AnonymizedApplication from "./schema/application/anonymizedApplications.schema"
import Application from "./schema/application/applications.schema"
import AppointmentDetailed from "./schema/appointmentDetailed/appointmentDetailed.schema"
import Appointment from "./schema/appointments/appointment.schema"
import Credential from "./schema/credentials/credential.schema"
import CustomEmailETFA from "./schema/customEmailETFA/customEmailETFA.schema"
import DiplomesMetiers from "./schema/diplomesmetiers/diplomesmetiers.schema"
import DomainesMetiers from "./schema/domainesmetiers/domainesmetiers.schema"
import EligibleTrainingsForAppointment from "./schema/eligibleTrainingsForAppointment/eligibleTrainingsForAppointment.schema"
import eligibleTrainingsForAppointmentHistory from "./schema/eligibleTrainingsForAppointmentsHistory/eligibleTrainingsForAppointmentHistory.schema"
import EmailBlacklist from "./schema/emailBlacklist/emailBlacklist.schema"
import Etablissement from "./schema/etablissements/etablissement.schema"
import FicheMetierRomeV4 from "./schema/ficheRomeV4/ficheRomeV4"
import FormationCatalogue from "./schema/formationCatalogue/formationCatalogue.schema"
import GeoLocation from "./schema/geolocation/geolocation.schema"
import InternalJobs from "./schema/internalJobs/internalJobs.schema"
import Job from "./schema/jobs/jobs.schema"
import LbaCompany from "./schema/lbaCompany/lbaCompany.schema"
import LbaCompanyLegacy from "./schema/lbaCompanylegacy/lbaCompanyLegacy.schema"
import { Cfa } from "./schema/multiCompte/cfa.schema"
import { Entreprise } from "./schema/multiCompte/entreprise.schema"
import { RoleManagement } from "./schema/multiCompte/roleManagement.schema"
import { UserWithAccount } from "./schema/multiCompte/userWithAccount.schema"
import Opco from "./schema/opco/opco.schema"
import Optout from "./schema/optout/optout.schema"
import Recruiter from "./schema/recruiter/recruiter.schema"
import RecruteurLbaUpdateEvent from "./schema/recruteurLbaUpdateEvent/recruteurLbaUpdateEvent.schema"
import ReferentielOnisep from "./schema/referentielOnisep/referentielOnisep.schema"
import ReferentielOpco from "./schema/referentielOpco/referentielOpco.schema"
import ReferentielRome from "./schema/referentielRome/referentielRome"
import Session from "./schema/session/session.schema"
import SiretDiffusibleStatus from "./schema/siretDiffusibleStatusSchema/siretDiffusibleStatusSchema.schema"
import UnsubscribedLbaCompany from "./schema/unsubscribedLbaCompany/unsubscribedLbaCompany.schema"
import UnsubscribeOF from "./schema/unsubscribedOF/unsubscribeOF.schema"
import User from "./schema/user/user.schema"

const createSpecialIndexes = async () => {
  await db.collection("bonnesboites").createIndex({ geopoint: "2dsphere" })
  await db.collection("formationcatalogues").createIndex({ lieu_formation_geopoint: "2dsphere" })
  await db.collection("recruiters").createIndex({ geopoint: "2dsphere" })
}

export async function createMongoDBIndexes() {
  const results = await Promise.allSettled(
    mongooseInstance.modelNames().map(async (name) => {
      const model = mongooseInstance.model(name)
      return model.createIndexes({ background: true }).catch(async (e) => {
        if (e.codeName === "IndexOptionsConflict" || e.codeName === "IndexKeySpecsConflict") {
          const err = new Error(`Conflict in indexes for ${name}`, { cause: e })
          logger.error(err)
          captureException(err)
          await mongooseInstance.connection.collection(model.collection.name).dropIndexes()
          await model.createIndexes({ background: true })
        } else {
          throw e
        }
      })
    })
  )

  await createSpecialIndexes()

  const errors = results.reduce((acc, r) => {
    if (r.status === "rejected") {
      acc.push(r.reason)

      logger.error(r.reason)
      captureException(r.reason)
    }

    return acc
  }, [] as Error[])

  if (errors.length > 0) {
    throw new AggregateError(errors, `createMongoDBIndexes failed with ${errors.length} errors`)
  }
}

export {
  AnonymizedApplication,
  AnonymizedUser,
  ApiCalls,
  Application,
  Appointment,
  AppointmentDetailed,
  Cfa,
  Credential,
  CustomEmailETFA,
  DiplomesMetiers,
  DomainesMetiers,
  EligibleTrainingsForAppointment,
  EmailBlacklist,
  Entreprise,
  Etablissement,
  FicheMetierRomeV4,
  FormationCatalogue,
  GeoLocation,
  InternalJobs,
  Job,
  LbaCompany,
  LbaCompanyLegacy,
  RecruteurLbaUpdateEvent,
  Opco,
  Optout,
  Recruiter,
  ReferentielOnisep,
  ReferentielOpco,
  ReferentielRome,
  RoleManagement,
  Session,
  SiretDiffusibleStatus,
  UnsubscribeOF,
  UnsubscribedLbaCompany,
  User,
  UserWithAccount,
  eligibleTrainingsForAppointmentHistory,
}
