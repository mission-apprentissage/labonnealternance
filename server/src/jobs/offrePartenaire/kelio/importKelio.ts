import { JOBPARTNERS_LABEL } from "shared/models/jobsPartners.model"
import rawKelioModel from "shared/models/rawKelio.model"

import config from "../../../config"
import { importFromStreamInXml } from "../importFromStreamInXml"
import { importFromUrlInXml } from "../importFromUrlInXml"
import { rawToComputedJobsPartners } from "../rawToComputedJobsPartners"

import { kelioJobToJobsPartners, ZKelioJob } from "./kelioMapper"

const rawCollectionName = rawKelioModel.collectionName
const offerXmlTag = "job"

export const importMonsterRaw = async (sourceStream?: NodeJS.ReadableStream) => {
  if (sourceStream) {
    await importFromStreamInXml({ destinationCollection: rawCollectionName, offerXmlTag, stream: sourceStream, partnerLabel: JOBPARTNERS_LABEL.MONSTER })
  } else {
    await importFromUrlInXml({ destinationCollection: rawCollectionName, url: config.monsterUrl, offerXmlTag, partnerLabel: JOBPARTNERS_LABEL.MONSTER })
  }
}

export const importMonsterToComputed = async () => {
  await rawToComputedJobsPartners({
    collectionSource: rawCollectionName,
    partnerLabel: JOBPARTNERS_LABEL.KELIO,
    zodInput: ZKelioJob,
    mapper: kelioJobToJobsPartners,
    documentJobRoot: offerXmlTag,
  })
}
