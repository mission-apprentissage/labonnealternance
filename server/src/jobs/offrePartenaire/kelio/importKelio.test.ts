import fs from "node:fs"

import { JOBPARTNERS_LABEL } from "shared/models/jobsPartners.model"
import { beforeEach, describe, expect, it, vi } from "vitest"

import { getDbCollection } from "@/common/utils/mongodbUtils"
import { useMongo } from "@tests/utils/mongo.test.utils"

import { importKelioRaw, importKelioToComputed } from "./importKelio"

const now = new Date("2024-07-21T04:49:06.000+02:00")

describe("importKelio", () => {
  useMongo()

  beforeEach(() => {
    vi.useFakeTimers()
    vi.setSystemTime(now)

    return async () => {
      vi.useRealTimers()
      await getDbCollection("computed_jobs_partners").deleteMany({})
      await getDbCollection("raw_hellowork").deleteMany({})
    }
  })

  it("should test the import of kelio data into computed_job_partners", async () => {
    const fileStream = fs.createReadStream("server/src/jobs/offrePartenaire/kelio/importKelio.test.input.xml")
    await importKelioRaw(fileStream)
    expect.soft(await getDbCollection("raw_kelio").countDocuments({})).toBe(2)

    await importKelioToComputed()
    const jobs = (
      await getDbCollection("computed_jobs_partners")
        .find({ partner_label: JOBPARTNERS_LABEL.KELIO }, { projection: { _id: 0, created_at: 0 } })
        .toArray()
    ).sort((a, b) => ((a.partner_job_id ?? "") < (b.partner_job_id ?? "") ? -1 : 1))
    expect.soft(jobs.length).toBe(2)
    expect.soft(jobs).toMatchSnapshot()
  })
})
