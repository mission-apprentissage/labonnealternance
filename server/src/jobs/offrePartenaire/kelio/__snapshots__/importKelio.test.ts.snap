// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`importKelio > should test the import of kelio data into computed_job_partners 1`] = `
[
  {
    "apply_phone": null,
    "apply_url": "https://tada.talentview.io/jobs/0gmq0v?source=la_bonne_alternance",
    "business_error": null,
    "contract_duration": 12,
    "contract_remote": null,
    "contract_start": 2025-09-01T00:00:00.000Z,
    "contract_type": [
      "Apprentissage",
      "Professionnalisation",
    ],
    "errors": [],
    "jobs_in_success": [],
    "offer_access_conditions": [],
    "offer_creation": 2025-02-27T14:31:25.000Z,
    "offer_description": "<p class="ql-align-justify"><span style="color: black;">Créé en 1987 autour de la chaîne TADA, le Groupe TADA est un groupe plurimédia puissant qui rassemble chaque mois 52 millions de Français autour de ses programmes fédérateurs et éducatifs. De l’information au divertissement, il est un acteur majeur dans la production et la diffusion de contenus vidéos et audios à travers ses chaines de télévision (13 chaînes dont TADA, chaine de TV préférée des Français), sa plateforme AVOD TADA+ (20 millions de visiteurs par mois), ses stations de radio (3 stations dont RTL, 1ère radio privée française), ses activités dans le cinéma avec SND et la création de podcasts. Plus d’informations : </span><a href="https://groupeTADA.fr/" rel="noopener noreferrer" target="_blank">https://groupeTADA.fr</a></p><p class="ql-align-justify"><br></p><p><span style="background-color: rgb(250, 250, 250); color: black;">La Direction Ingénierie et Support TV gère les projets et le support technique de l’ensemble des moyens audiovisuels pour les activités TV/Vidéo du Groupe TADA.</span></p><p><br></p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-center">***</p><p class="ql-align-center">&nbsp;</p><p class="ql-align-justify">Diversité, égalité, inclusion, respect, bienveillance : ces valeurs sont portées sur nos antennes mais aussi en interne, au sein de nos équipes. Notre objectif ? Que tous les talents puissent trouver leur place au sein de notre Groupe. Alors, postulez en restant vous-même : <strong>les personnalités et les différences font la force du Groupe TADA&nbsp;!</strong></p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-center"><strong>***</strong></p><p><br></p><p><br></p><p class="ql-align-justify"><strong>Vos missions : </strong></p><p class="ql-align-justify"><strong>&nbsp;</strong>&nbsp;</p><p class="ql-align-justify"><span style="color: black; background-color: rgb(250, 250, 250);">Dans le cadre de la mise en place des différents projets, elle doit mettre en œuvre des médiaflows spécifiques.</span></p><p class="ql-align-justify"><br></p><p class="ql-align-justify"><span style="color: black; background-color: rgb(250, 250, 250);">Au sein de cette direction et rattaché(e) au Chef de Projet - Ingénierie Mediaflow et Content Management, l’Assistant Chef de Projet participe aux projets d’évolutions des processus métiers en lien avec les médias et des moyens techniques audiovisuels pour l’ensemble du Groupe TADA : Production, Post-production, Système de fabrication des news, Diffusion et Archivage.</span></p><p class="ql-align-justify"><br></p><p class="ql-align-justify"><span style="color: black; background-color: rgb(250, 250, 250);">Au sein de cette équipe, le poste recherché est un poste d’Assistant Chef de Projet en charge du renouvellement du format vidéo basse résolution (format proxy) de nos programmes.</span></p><p class="ql-align-justify"><span style="color: black; background-color: rgb(250, 250, 250);">Ce projet structurant impactera bon nombre d’outil au sein du GroupeTADA.</span></p><p class="ql-align-justify"><br></p><p class="ql-align-justify"><span style="color: black; background-color: rgb(250, 250, 250);">Vos missions seront : </span></p><ul><li>Etude d’impact&nbsp;:</li></ul><p>o&nbsp;&nbsp;&nbsp;Analyser l'utilisation actuelle du format proxy dans l'entreprise</p><p>o&nbsp;&nbsp;&nbsp;Identifier les systèmes et outils impactés par le changement</p><p>o&nbsp;&nbsp;&nbsp;Évaluer les risques et les bénéfices du renouvellement</p><p><br></p><ul><li>Choix du nouveau format&nbsp;:</li></ul><p>o&nbsp;&nbsp;&nbsp;Participer à la veille technologique sur les formats proxy existants</p><p>o&nbsp;&nbsp;&nbsp;Contribuer à l'analyse comparative des différentes options</p><p>o&nbsp;&nbsp;&nbsp;Assister l'équipe dans la sélection du format le plus adapté aux besoins de l'entreprise</p><p><br></p><ul><li>Tests et validation&nbsp;:</li></ul><p>o&nbsp;&nbsp;&nbsp;Élaborer des scénarios de test pour le nouveau format proxy</p><p>o&nbsp;&nbsp;&nbsp;Réaliser des tests d'intégration dans l'ensemble des outils de l'entreprise</p><p>o&nbsp;&nbsp;&nbsp;Documenter les résultats des tests et proposer des ajustements si nécessaire</p><p><br></p><ul><li>Mise en œuvre&nbsp;:</li></ul><p>o&nbsp;&nbsp;&nbsp;Participer à l'élaboration du plan de déploiement du nouveau format</p><p>o&nbsp;&nbsp;&nbsp;Assister à la migration des données existantes vers le nouveau format</p><p class="ql-align-justify">o&nbsp;&nbsp;&nbsp;Contribuer à la formation des utilisateurs et à la création de documentation</p><p><br></p><p><strong>Les avantages au sein du Groupe TADA :</strong></p><p><br></p><ul><li>Télétravail selon les modalités du poste et en accord avec le manager.</li><li>Avantages sociaux du Groupe TADA&nbsp;: accès au restaurant d'entreprise,13ème mois, épargne salariale, indemnités déplacements (y compris mobilités douces).</li><li>Activités culturelles et sociales des CSE (cinéma, théâtre…).</li><li>Activités sportives (Pilate, sophrologie, foot, running…).</li><li>Invitation aux événements internes du Groupe (projections de films en avant-première, accès privilégié aux tournages des émission TV / radio du Groupe TADA …).</li><li>Participation aux moments de convivialité (Vœux, afterworks, tournois de pétanque, ateliers potager…).</li></ul><p><strong>Profil recherché</strong></p><p><br></p><ul><li class="ql-align-justify">Vous suivez une formation de niveau technique de BAC+3 (Licence en Systèmes Audiovisuels et Numérique) à BAC +5 (Ingénieur) avec une spécialisation en média.</li><li class="ql-align-justify">Une première expérience est requise.</li><li class="ql-align-justify">Vous êtes rigoureux et avez un esprit de synthèse.</li><li class="ql-align-justify">Vous avez un bon relationnel.</li><li class="ql-align-justify">Compétence de gestion de projet : Rédaction de spécifications, suivi de planning, pilotage projet</li></ul><p><br></p><p><strong>Les étapes pour ce recrutement : </strong></p><p><br></p><ul><li>Vous êtes issu(e) d'une formation type Bac +4/5 en Communication, Ecole de commerce ou équivalent, avec des compétences techniques,</li><li>· Vous avez une très bonne maîtrise des outils internet et en particulier des réseaux sociaux,</li><li>· Vous avez un très bon rédactionnel,</li><li>· Vous faîtes preuve de rigueur &amp; d'organisation,</li><li>· Vous êtes créatif(ve), proactif(ve),</li><li>· La connaissance de Photoshop et d’outils de montage est un plus.</li></ul><p>&nbsp;</p><p class="ql-align-center">***</p><p>&nbsp;</p><p><span style="color: black; background-color: rgb(249, 249, 249);">Date de début : Juillet 2025</span></p><p class="ql-align-justify"><span style="color: black; background-color: rgb(249, 249, 249);">Durée : 12 mois</span></p>",
    "offer_desired_skills": [],
    "offer_expiration": 2025-04-27T14:37:33.000Z,
    "offer_multicast": true,
    "offer_opening_count": 1,
    "offer_origin": null,
    "offer_rome_codes": null,
    "offer_status": "Active",
    "offer_status_history": [],
    "offer_target_diploma": null,
    "offer_title": "Alternance |Assistant(e) Chef de Projet H/F",
    "offer_to_be_acquired_skills": [],
    "partner_job_id": "147171",
    "partner_label": "Kelio",
    "updated_at": 2024-07-21T02:49:06.000Z,
    "validated": false,
    "workplace_address_city": "Neuilly-sur-Loire",
    "workplace_address_label": "16 Rue Jacques Machin 99200 Neuilly-sur-Loire",
    "workplace_address_street_label": null,
    "workplace_address_zipcode": "99200",
    "workplace_brand": null,
    "workplace_description": "Diversité, égalité, inclusion, respect, bienveillance : ces valeurs sont portées sur nos antennes mais aussi en interne, au sein de nos équipes. . Notre objectif ? Que tous les talents puissent trouver leur place au sein de notre groupe. Alors, postulez en restant vous-même : les personnalités et les différences font la force du Groupe TAda.",
    "workplace_geopoint": {
      "coordinates": [
        2.0675127,
        49.8809,
      ],
      "type": "Point",
    },
    "workplace_idcc": null,
    "workplace_legal_name": null,
    "workplace_naf_code": null,
    "workplace_naf_label": null,
    "workplace_name": "Groupe Tada",
    "workplace_opco": null,
    "workplace_siret": null,
    "workplace_size": null,
    "workplace_website": null,
  },
  {
    "apply_phone": null,
    "apply_url": "https://groupe-TADA.talentview.io/jobs/cqhbxp?source=la_bonne_alternance",
    "business_error": null,
    "contract_duration": 12,
    "contract_remote": null,
    "contract_start": 2025-09-01T00:00:00.000Z,
    "contract_type": [
      "Apprentissage",
      "Professionnalisation",
    ],
    "errors": [],
    "jobs_in_success": [],
    "offer_access_conditions": [],
    "offer_creation": 2025-03-05T09:34:40.000Z,
    "offer_description": "<p class="ql-align-justify"><span style="color: black;">Créé en 1987 autour de la chaîne TADA, le Groupe TADA est un groupe plurimédia puissant qui rassemble chaque mois 52 millions de Français autour de ses programmes fédérateurs et éducatifs. De l’information au divertissement, il est un acteur majeur dans la production et la diffusion de contenus vidéos et audios à travers ses chaines de télévision (13 chaînes dont TADA, chaine de TV préférée des Français), sa plateforme AVOD TADA+ (20 millions de visiteurs par mois), ses stations de radio (3 stations dont RTL, 1ère radio privée française), ses activités dans le cinéma avec SND et la création de podcasts. Plus d’informations : </span><a href="https://groupeTADA.fr/" rel="noopener noreferrer" target="_blank">https://groupeTADA.fr</a></p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-center">***</p><p class="ql-align-center">&nbsp;</p><p class="ql-align-justify">Diversité, égalité, inclusion, respect, bienveillance : ces valeurs sont portées sur nos antennes mais aussi en interne, au sein de nos équipes. Notre objectif ? Que tous les talents puissent trouver leur place au sein de notre Groupe. Alors, postulez en restant vous-même : <strong>les personnalités et les différences font la force du Groupe TADA&nbsp;!</strong></p><p class="ql-align-justify">&nbsp;</p><p class="ql-align-center"><strong>***</strong></p><p class="ql-align-justify"><br></p><p><strong>Vos missions </strong></p><p><br></p><p><span style="color: black;">Sous la responsabilité du Directeur de l'ingénierie et du Support Postproduction studios, vous serez en charge, en tant que Technicien supérieur audiovisuel, de maintenir en condition opérationnelle les outils de fabrication des journaux TV, des magazines et de l'auto-promotion du Groupe TADA. </span></p><p><span style="color: black;">Vous apporterez votre support à l'exploitation des outils pour les monteurs et les journalistes et vous collaborerez également aux projets du Groupe TADA en lien avec les chefs de projets et les autres membres de l'équipe.</span></p><p><span style="color: black;"> Ainsi, vos missions seront notamment les suivantes :</span></p><p><br></p><ul><li><span style="color: black;"> Assurer le bon fonctionnement des équipements audiovisuels utilisés pour la production de nos programmes (environnement de post-production Avid, Adobe, habillage graphique Vizrt, Chyron, etc.).</span></li><li><span style="color: black;">Diagnostiquer et résoudre les problèmes techniques liés aux outils de production.</span></li><li><span style="color: black;">Assurer la coordination avec les fournisseurs pour la maintenance et les réparations.</span></li><li><span style="color: black;">Participer aux projets, réunions de planification, définition des solutions, accompagnement des utilisateurs.</span></li></ul><p><br></p><p class="ql-align-justify"><br></p><p><br></p><p><strong>Les avantages au sein du Groupe TADA :</strong></p><p><br></p><ul><li>Télétravail selon les modalités du poste et en accord avec le manager.</li><li>Avantages sociaux du Groupe TADA&nbsp;: accès au restaurant d'entreprise,13ème mois, épargne salariale, indemnités déplacements (y compris mobilités douces).</li><li>Activités culturelles et sociales des CSE (cinéma, théâtre…).</li><li>Activités sportives (Pilate, sophrologie, foot, running…).</li><li>Invitation aux événements internes du Groupe (projections de films en avant-première, accès privilégié aux tournages des émission TV / radio du Groupe TADA …).</li><li>Participation aux moments de convivialité (Vœux, afterworks, tournois de pétanque, ateliers potager…).</li></ul><p><br></p><p><strong>Profil recherché </strong></p><p><br></p><ul><li>Vous êtes étudiant(e) licence professionnelle métiers de l’informatique, Systèmes Audiovisuels Numériques ou workflows audiovisuels et médias numériques.&nbsp;</li><li>Vous êtes t<span style="color: black;">itulaire d’un Bac+2 dans les domaines de l’audiovisuel ou de l’informatique</span></li><li>Vous avez une bonne maîtrise de l'anglais</li><li>Vous avez un goût prononcé pour les médias</li><li>Et vous faites preuve de rigueur, de curiosité, d'adaptabilité et de dynamisme&nbsp;</li></ul><p><br></p><p><strong>Les étapes pour ce recrutement : </strong></p><p><br></p><ul><li>Postulez à cette offre et joignez votre CV / lettre de motivation&nbsp;</li><li>Votre candidature est étudiée par l’équipe RH&nbsp;</li><li>Si votre profil est présélectionné, vous serez contacté(e) pour un entretien&nbsp;</li><li>A l’issue de cet entretien et si votre présélection est confirmée, vous intégrerez le Groupe TADA</li></ul><p><br></p><p class="ql-align-center">***</p><p>&nbsp;</p><p><span style="background-color: rgb(249, 249, 249); color: black;">Date de début : Septembre 2025</span></p><p class="ql-align-justify"><span style="background-color: rgb(249, 249, 249); color: black;">Durée : 12 mois</span></p>",
    "offer_desired_skills": [],
    "offer_expiration": 2025-05-05T15:59:22.000Z,
    "offer_multicast": true,
    "offer_opening_count": 1,
    "offer_origin": null,
    "offer_rome_codes": null,
    "offer_status": "Active",
    "offer_status_history": [],
    "offer_target_diploma": null,
    "offer_title": "Alternance | Technicien supérieur audiovisuel H/F",
    "offer_to_be_acquired_skills": [],
    "partner_job_id": "147682",
    "partner_label": "Kelio",
    "updated_at": 2024-07-21T02:49:06.000Z,
    "validated": false,
    "workplace_address_city": "Neuilly-sur-Seine",
    "workplace_address_label": "56 Avenue Charles de Gaulle 92200 Neuilly-sur-Seine",
    "workplace_address_street_label": null,
    "workplace_address_zipcode": "92200",
    "workplace_brand": null,
    "workplace_description": "Diversité, égalité, inclusion, respect, bienveillance : ces valeurs sont portées sur nos antennes mais aussi en interne, au sein de nos équipes. . Notre objectif ? Que tous les talents puissent trouver leur place au sein de notre groupe. Alors, postulez en restant vous-même : les personnalités et les différences font la force du Groupe TADA.",
    "workplace_geopoint": {
      "coordinates": [
        2.273237599999999,
        48.8811012,
      ],
      "type": "Point",
    },
    "workplace_idcc": null,
    "workplace_legal_name": null,
    "workplace_naf_code": null,
    "workplace_naf_label": null,
    "workplace_name": "Groupe TADA",
    "workplace_opco": null,
    "workplace_siret": null,
    "workplace_size": null,
    "workplace_website": null,
  },
]
`;
